.github/workflows/Connector Master SDK Workflow.yml

```diff
@@ -82,6 +82,33 @@ jobs:
         run: dotnet test "${{ steps.findSlnFile.outputs.solutionFilePath }}" --filter TestCategory!=IntegrationTest --logger "trx;logfilename=unitTestResults.trx" --collect "XPlat Code Coverage" -- DataCollectionRunSettings.DataCollectors.DataCollector.Configuration.Format=cobertura,opencover
         continue-on-error: true
 
+      - name: Install Connector Validator
+        run: dotnet tool install -g Skyline.DataMiner.CICD.Tools.Validator
+  
+      - name: Run Connector Validator
+        run: dataminer-validator validate-protocol-solution --solution-path "${{ github.workspace }}" --output-directory "${{ github.workspace }}" --output-file-name "validateResults" 
+        continue-on-error: true
+
+      - name: Archive Results
+        uses: actions/upload-artifact@v4
+        with:
+          name: validateResults
+          path: ${{ github.workspace }}/validateResults.json
+        continue-on-error: true
+
+      - name: Validator Quality Gate
+        id: validator-quality-gate
+        run: |
+          json=$(cat "${{ github.workspace }}/validateResults.json")
+          critical=$(echo "$json" | jq -r '.CriticalIssueCount')
```

Comment:
use fromJson() instead?

Reply:
Code generated from a language model, works as it should. the "jq" indicates it parses it as json to find the right field.

Reply:
can the code be updated so that the if clause below compares numeric values instead of string "0"?

Reply:
The bash code was adjusted.

Reply:
Resolved in person. Good to go.